lib/ui/platform_dispatcher.dart

```diff
@@ -1120,6 +1131,9 @@ class ViewConfiguration {
   /// phone sensor housings).
   final WindowPadding padding;
 
+  /// The view specific gesture settings.
```

Comment:
nit: a little more information in the docs on what one can expect here could be beneficial.

Reply:
Added more docs